### PR TITLE
workflows/tests: run tap_syntax on master push

### PR DESCRIPTION
**Pull request by Bo98**
Disclaimer: I haven't tested this

### CODE REVIEW

**Comment by Bo98** [comment]
That didn't work apparently.

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -1,5 +1,8 @@
 name: GitHub Actions CI
 on:
+  push:
+    branches:
+      - master
```
Comment (MikeMcQuaid) [inline .github/workflows/tests.yml:5]:
```suggestion
    branches: master
```

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -46,6 +49,7 @@ jobs:
 
   tests:
     needs: tap_syntax
+    if: github.event_name == 'pull_request'
```
Comment (MikeMcQuaid) [inline .github/workflows/tests.yml:52]:
```suggestion
    if: github.ref != 'refs/heads/master'
```

**Review by MikeMcQuaid — COMMENTED** [review]
Try this:

**Review by MikeMcQuaid — APPROVED** [review]

**Comment by Bo98** [comment]
Tried applying one at the time and it seems the first one worked. Genuinely no idea why.

**Pull request merged**